ecs-agent/metrics/constants.go

```diff
@@ -46,6 +46,11 @@ const (
 	ACSDisconnectTimeoutMetricName = agentAvailabilityNamespace + ".ACSDisconnectTimeout"
 	TCSDisconnectTimeoutMetricName = agentAvailabilityNamespace + ".TCSDisconnectTimeout"
 
+	// ACS Session Metrics
+	acsStartSessionNamespace            = "ACSStartSession"
+	ACSDiscoverPollEndpointDurationName = acsStartSessionNamespace + ".DiscoverPollEndpointDuration"
```

Comment:
Why are we calling these two duration metrics? What duration is being measured?

Reply:
Clarified offline, these metrics include a duration by default which will tell us the connection duration.